src/vdaf/poplar1.rs

```diff
@@ -394,9 +399,9 @@ impl Aggregatable for Poplar1FieldVec {
 //
 // TODO(cjpatton) spec: Make sure repeated prefixes are disallowed. To make this check easier,
 // consider requring the prefixes to be in lexicographic order.
```

Comment:
The checks on L439 suggest this spec change did happen. Let's delete this comment.